refactored_all_teachers.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'school'
```

Comment:
a school is inanimate - can a human being inherit from a non-living thing? Naming is weird